src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -97,18 +97,172 @@ async function pickProject(
     transformByQState.setProjectName(encodeHTML(state.project.label)) // encode to avoid HTML injection risk
 }
 
+let intervalId = undefined
```

Comment:
I still have to refactor these variables. I move them as "globals" but deciding on using them as params or adding them to our object/store/singleton state.

Thoughts welcome!

Reply:
It does seem to suggest that we could encapsulate this into a class - `TransformationJob` (for example.) I'd suggest overall moving towards instance variables instead of global/sequence as it would get us closer to letting customers run multiple transformation jobs at once

Reply:
I moved the variables into the transformByQState.